Formula/hardlink.rb

```diff
@@ -21,6 +21,10 @@ class Hardlink < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "b849b6cdc2d96380221c9dddc48a4c6485db0f4257ea7c48ade59b50e29f7bfd"
   end
 
+  keg_only "has been merged into `util-linux`"
```

Comment:
Probably should put this inside an `on_linux` block since we still can't build `util-linux`'s hardlink on macOS. Due to stanza order, it will need to be moved below `depends_on`